NextcloudTalk/FileMessageTableViewCell.m

```diff
@@ -206,24 +218,43 @@ - (void)setupForMessage:(NCChatMessage *)message withLastCommonReadMessage:(NSIn
                                        dispatch_async(dispatch_get_main_queue(), ^(void){
                                            CGFloat width = image.size.width * image.scale;
                                            CGFloat height = image.size.height * image.scale;
-                                           if (height > kFileMessageCellFilePreviewHeight) {
-                                               CGFloat ratio = kFileMessageCellFilePreviewHeight / height;
+                                           
+                                           CGFloat previewHeight = isMediaFile ? kFileMessageCellMediaFilePreviewHeight : kFileMessageCellFilePreviewHeight;
+                                           CGFloat previewMaxWidth = isMediaFile ? kFileMessageCellMediaFileMaxPreviewWidth : kFileMessageCellFileMaxPreviewWidth;
+                                           
+                                           if (height > previewHeight) {
+                                               CGFloat ratio = previewHeight / height;
                                                width = width * ratio;
-                                               height = kFileMessageCellFilePreviewHeight;
+                                               height = previewHeight;
                                            }
-                                           if (width > maxPreviewImageWidth) {
-                                               CGFloat ratio = maxPreviewImageWidth / width;
-                                               width = maxPreviewImageWidth;
+                                           if (width > previewMaxWidth) {
+                                               CGFloat ratio = previewMaxWidth / width;
+                                               width = previewMaxWidth;
                                                height = height * ratio;
                                            }
                                            weakSelf.vPreviewSize[3].constant = height;
                                            weakSelf.hPreviewSize[3].constant = width;
                                            weakSelf.vGroupedPreviewSize[1].constant = height;
                                            weakSelf.hGroupedPreviewSize[1].constant = width;
+                                           // if the image is very narrow, use a very small corner radius
+                                           if (height < 2 * kFileMessageCellMediaFilePreviewCornerRadius || width < 2 * kFileMessageCellMediaFilePreviewCornerRadius) {
+                                               weakSelf.previewImageView.layer.cornerRadius = MIN(height, width) / 2.0;
+                                           }
+                                           // use a bigger corner radius for media file previews since they're shown bigger
```

Comment:
Please put the comment inside the `else` (might want to do it with the `if` one as well), so we have `} else {`